bee-message/src/output/alias.rs

```diff
@@ -61,8 +54,12 @@ impl AliasOutputBuilder {
         byte_cost_config: ByteCostConfig,
         alias_id: AliasId,
     ) -> Result<AliasOutputBuilder, Error> {
+        Self::new(OutputBuilderAmount::MinimumStorageDeposit(byte_cost_config), alias_id)
+    }
+
+    fn new(amount: OutputBuilderAmount, alias_id: AliasId) -> Result<AliasOutputBuilder, Error> {
```

Comment:
Could something like
`fn new(amount: impl Into<OutputBuilderAmount>, alias_id: AliasId)` make the API a bit simpler?

Reply:
It's a private method, it doesn't really need to be simpler than that